run-e2e-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+cd e2e
+
+npm install
```

Comment:
Thoughts on extracting this to `install-e2e-test.sh`?

Reply:
I'd rather leave it as it is because this must be invoked using node `14`. Once we manage to run everything using a single node version I will extract this.